### ISSUE TITLE
[Package]: DevkitARM

### ISSUE THREAD

**Issue by Chhayank-Sharma** [termux/termux-packages#7960]
### Package description

devkitARM is a tool based on the gcc compiler which is used to make gba and nds games. it is part of the devkitPro toolchain. it is ussually installed via the pacman package manager but it is not available in termux. please add it. heres a link to thier
wiki: [https://devkitpro.org/wiki/Main_Page](url)
github main page: [https://github.com/devkitPro](url)
build scripts: [https://github.com/devkitPro/buildscripts](url)

### Home page URL

https://devkitpro.org/wiki/Main_Page

### Source code URL

https://github.com/devkitPro

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

**Comment by Grimler91** [comment]
We don't even have gcc, and it is highly unlikely that any of the core-contributors are going to spend time trying to port it.

It would be more realistic to look for some gba/nds compiler based on clang

**Comment by Maxython** [comment]
> It would be more realistic to look for some gba/nds compiler based on clang

Or configure it to work with clang.

**Comment by Chhayank-Sharma** [comment]
> We don't even have gcc, and it is highly unlikely that any of the core-contributors are going to spend time trying to port it.
> 
> It would be more realistic to look for some gba/nds compiler based on clang

gcc can already be used in termux using third party repositories. also, the reason why I want to use devkitARM is because it is the most popular and most of the tutorials including the one i am following are for devkitARM only. i am using tonc to learn gba development btw.

**Comment by Grimler91** [comment]
> gcc can already be used in termux using third party repositories

Yes, that's true. Sounds like this package request would be more appropriate in the same third-party repository then

We can't add gcc from the third-party repository to the main repository, as there are no (cross-compilation) build recipes available (gcc from that repo has been cross-compiled using an older NDK version, or compiled on device)

**Comment by Grimler91** [comment]
See also https://github.com/termux/termux-packages/issues/407, https://github.com/termux/termux-packages/issues/767, https://github.com/termux/termux-packages/issues/2511
